CHANGES/1169.packaging.rst

```diff
@@ -0,0 +1,4 @@
+Switched to using the :mod:`propcache` package for property caching -- by :user:`bdraco`.
+
+The :mod:`propcache` package is derived from the property caching code in :mod:`yarl`
+and has been broken out to avoid maintaining it for multiple projects.
```

Comment:
```suggestion
Switched to using the :mod:`propcache <propcache.api>` package for property caching
-- by :user:`bdraco`.

The :mod:`propcache <propcache.api>` package is derived from the property caching
code in :mod:`yarl` and has been broken out to avoid maintaining it for multiple
projects.
```